ui/src/app/archived-workflows/components/archived-workflow-list/archived-workflow-list.tsx

```diff
@@ -62,42 +56,32 @@ export class ArchivedWorkflowList extends BasePage<RouteComponentProps<any>, Sta
     }
 
     public render() {
-        if (this.state.loading) {
```

Comment:
move error shims lower in the dom, so more the page remains responsive, you'll see this several times in this PR